novawallet/Modules/Vote/Governance/Referendums/Model/ReferendumsViewState.swift

```diff
@@ -0,0 +1,8 @@
+struct ReferendumsViewState: Equatable {
+    var cells: [ReferendumsCellViewModel]
+    var timeModels: [ReferendumIdLocal: StatusTimeViewModel?]?
+}
+
+struct ReferendumsState {
+    var referendums: [ReferendumLocal]
```

Comment:
What about account voting? And Why it is var and not let?

Reply:
The model is not in use. There were plans for it, but not anymore.